web-www/components/templates/Layout/Layout.config.ts

```diff
@@ -100,7 +100,7 @@ export const footerItems: [FooterItem, FooterItem, FooterItem, FooterItem] = [
       },
       {
         title: 'Community Calendar',
-        href: 'https://calendar.google.com/calendar/u/0/embed?src=c_mldg58l0nb1g22sb7treh6s63s@group.calendar.google.com&ctz=America/Denver',
+        href: 'https://calendar.google.com/calendar/u/0/embed?src=c_1250640b0093c1453ac648937f168c97e48175e7862fb02d38f45e0639df2b25@group.calendar.google.com&ctz=America/New_York',
```

Comment:
```suggestion
        href: 'https://calendar.google.com/calendar/u/0/embed?src=c_1250640b0093c1453ac648937f168c97e48175e7862fb02d38f45e0639df2b25@group.calendar.google.com',
```

Reply:
Good catch. I tried to remove it from the URL but the calendar is still using the US timezone. We might need an other link here.
/cc @erikalogie

Reply:
This is working for me if I open another browser. Maybe loading the same timezone due to local cache? One browser is loading the local timezone and the other is loading the NY timezone. :shrug:

Reply:
You're right I think it was a caching issue.